libraries/exoplayer_dash/src/main/java/androidx/media3/exoplayer/dash/offline/DashDownloader.java

```diff
@@ -113,13 +113,32 @@ public DashDownloader(
    * @param executor An {@link Executor} used to make requests for the media being downloaded.
    *     Providing an {@link Executor} that uses multiple threads will speed up the download by
    *     allowing parts of it to be executed in parallel.
+   * @param maxMergedSegmentStartTimeDiffUs max merged size for each segment duration
    */
+  public DashDownloader(
+      MediaItem mediaItem,
+      Parser<DashManifest> manifestParser,
+      CacheDataSource.Factory cacheDataSourceFactory,
+      Executor executor,
+      long maxMergedSegmentStartTimeDiffUs
+  ) {
+    super(mediaItem, manifestParser, cacheDataSourceFactory, executor, maxMergedSegmentStartTimeDiffUs);
+    baseUrlExclusionList = new BaseUrlExclusionList();
+  }
+
+  @Deprecated
```

Comment:
nit: We have a standardized header for this which is 

```
  /**
   * @deprecated Use {@link DashDownloader#DashDownloader(MediaItem, Parser,
   *       CacheDataSource.Factory, Executor, long)} instead.
   */
  @Deprecated
```

we need both annotation inside and outside the comment.

When at it, can you please move the new constructor to the end of the list of constructors (most params at the bottom)?


ditto for HlsDownloader and SsDownloader.

Reply:
will do!